bridgeql/django/settings.py

```diff
@@ -84,7 +84,8 @@ def validate(self):
             self._validate_auth_decorator()
         )
 
-    def get_local_apps(self):
+    @classmethod
+    def get_local_apps(cls):
         local_apps = []
         project_root = os.path.abspath(settings.BASE_DIR)
```

Comment:
django 2.0+ implements BASE_DIR where as 1.x implements SITE_ROOT
Should we fallback to SITE_ROOT if unable to get BASE_DIR ?